jwst/regtest/test_nircam_tsgrism.py

```diff
@@ -69,3 +73,30 @@ def test_nircam_tsgrism_stage3_whtlt(run_pipelines):
 
     # setdiff returns a table of length zero if there is no difference
     assert len(setdiff(table, table_truth)) == 0
+
+
+@pytest.mark.bigdata
+def test_nircam_setpointing(_jail, rtdata, fitsdiff_default_kwargs):
```

Comment:
Why using `_jail` instead of `jail`?

Reply:
If one wants function-scoped working directory, `_jail` and `rtdata` fixtures are appropriate.  Maybe we should rename these to something more obvious?